contracts/HolographOperator.sol

```diff
@@ -294,7 +297,9 @@ contract HolographOperator is Admin, Initializable, IHolographOperator {
       // decrease pod size to accomodate popped operator
       _operatorTempStorage[_operatorTempStorageCounter] = _operatorPods[pod][operatorIndex];
       _popOperator(pod, operatorIndex);
-      podSize--;
+      if (podSize > 1) {
```

Comment:
This to keep it as just one pod for now?

Reply:
This is to prevent an error for the random math if there is only one actively bonded operator in the pod, which at this point would be a zero address, allowing anyone to operate. Otherwise we would have the `_RBH` command fail since a zero is used in the math calculations.

Reply:
ahh